.github/workflows/terraformApply.yml

```diff
@@ -83,3 +87,91 @@ jobs:
         run: "TF_LOG=info terraform apply -auto-approve -input=false ${{ env.terraform_directory }}"
         env:
           VAULT_TOKEN: ${{ steps.token.outputs.vault_token }}
+  notify_deploy_start:
+    name: Announce deployments starting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch Slack token
+        id: vault-lookup
+        uses: hashicorp/vault-action@v2.1.2
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.MONSTER_AUTH_ROLE_ID }}
+          secretId: ${{ secrets.MONSTER_AUTH_ROLE_SECRET }}
+          secrets: secret/dsde/monster/dev/slack-notifier oauth-token | slack_token
+          exportEnv: false
+      - name: Send Slack notification
+        id: notification
+        uses: voxmedia/github-action-slack-notify-build@v1
+        with:
+          channel_id: C01QNFFKAG2  # monster-deploy
```

Comment:
❓ Why not use `channel: monster-deploy` instead of the ID?

Reply:
slack's api (or at least this particular action) can't post directly to a channel by name, so if you provide the name instead, it needs to do a scan of all of the server's channels to find the id to post to. and broad slack has a _lot_ of channels.